src/global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "8.0.204",
+        "version": "8.0.10",
```

Comment:
am i missing something with .10?  that's a pre release version?

Reply:
"rollForward": "latestMinor" brings it to the latest available, I have had issues with using the latest version as sometimes even though its released the Github Build server fails to get it causing failure

Reply:
Historically we just specified "version": "8.0" this caused an issue when using DotNet Build, MsBuild doesnt care and all is well, now that we are switching away from MsBuild its more important. "version": "8.0.10" is enough detail for it to be happy and roll forward